mars/services/context.py

```diff
@@ -38,11 +39,13 @@ class ThreadedServiceContext(Context):
     def __init__(self,
                  session_id: str,
                  supervisor_address: str,
+                 worker_address: str,
```

Comment:
Is `worker_address` identical to `current_address`?

Reply:
They are not identical. Thus we need the extra one.

`current_address` is the address of the actor.